projects/wave-core-new/src/lib/backend/operator.model.ts

```diff
@@ -126,6 +126,18 @@ export interface PointInPolygonFilterDict extends OperatorDict {
     };
 }
 
+export interface ColumnRangeFilterDict extends OperatorDict {
+    type: 'ColumnRangeFilter',
+    params: {
+        column: string,
+        ranges: Array<number[]>,
```

Comment:
Ranges can be numbers or strings

Reply:
changed to `ranges: Array<number[]> | Array<string[]>,`